api/controllers/user_controller.go

```diff
@@ -56,6 +57,12 @@ func (u UserController) GetUser(c *gin.Context) {
 
 // SaveUser saves the user
 func (u UserController) SaveUser(c *gin.Context) {
+	// user := models.User{
+	// 	Age:   1,
+	// 	Email: "dineshsilwal@gmail.com",
+	// 	Name:  "Dinesh Silwal",
+	// }
+
```

Comment:
@Denes-cilwal Can you remove this comment ?

Reply:
@dipeshdulal Sir sure ..

Reply:
> @Denes-cilwal Can you remove this comment ?

sure @dipeshdulal  sir..